Azure/sonic-buildimage
# Migrate a7050-qx-32s to use the new buffers config architecture

Create the profile for t1:

Move the ingress lossless profile from buffers.json.j2 to pg_profile_lookup.ini

Move pool and the rest of the profile from buffers.json.j2 to
buffers_defaults_t1.j2

Add port speed info in port_config.ini

Make buffers_default_t1.j2 the default profile in buffers.json.j2

prepared with support from Ying

Signed-off-by: Wenda Ni <wenni@microsoft.com>



**- What I did**

**- How I did it**

**- How to verify it**
Load on dut and see if the BUFFER_PROFILE and BUFFER_POOL generated has the same config content in config_db.json as before.

redis-cli -n 4 KEYS "BUFFER_PG|Ethernet8|3-4" has loaded the new config format
1) "BUFFER_PG|Ethernet8|3-4"

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
